cpp/src/arrow/acero/CMakeLists.txt

```diff
@@ -221,36 +221,21 @@ if(ARROW_BUILD_BENCHMARKS)
 
   add_arrow_acero_benchmark(aggregate_benchmark SOURCES aggregate_benchmark.cc)
 
-  if(ARROW_BUILD_OPENMP_BENCHMARKS)
-    find_package(OpenMP REQUIRED)
```

Comment:
Feeling comfortable of removing one dependency.

Reply:
I think so too!